src/index.tsx

```diff
@@ -17,6 +21,40 @@ const ExpoMoquette = NativeModules.ExpoMoquette
       }
     );
 
-export function startServerAsync(a: number, b: number): Promise<number> {
-  return ExpoMoquette.startServerAsync(a, b);
+type ServerConfig = {
+  host: string;
+  port: string;
+  wssPort: string;
+};
+export function startServerAsync(config: ServerConfig): Promise<string> {
+  return ExpoMoquette.startServerAsync(config);
+}
+
+export function getConnectedList(): Promise<any> {
+  return ExpoMoquette.getConnectedClientsAsync();
+}
+
+export function restartServerAsync(): Promise<{ ok: Boolean }> {
+  return ExpoMoquette.restartServerAsync();
+}
+
+export function stopServerAsync(): Promise<{ ok: Boolean }> {
+  return ExpoMoquette.stopServerAsync();
+}
+
+export type ServerStatus = {
+  port: number;
+  sslPort: number;
+};
+
+export function getServerStatusAsync(): Promise<ServerStatus> {
+  return ExpoMoquette.getServerStatusAsync();
+}
+
+export function isAvailable() {
+  return Platform.OS === 'android';
+}
+const emitter = new NativeEventEmitter();
+export function subscribeTopic(topic: string, callback: (event: any) => void) {
+  return emitter.addListener(topic, callback);
```

Comment:
Cái topic chỗ này có thể dùng regexp/ hoặc xử lý prefix để filter
```suggestion
export function subscribeTopic(topic: string, callback: (event: any) => void) {
  return emitter.addListener('ON_MESSAGE', callback);
```